src/components/Button/Button.tsx

```diff
@@ -42,13 +42,15 @@ export const Button = ({
       <Icon
         name={iconLeft}
         size="small"
+        className="icon-left"
       />
     )}
     {label ? label : children}
     {iconRight && (
       <Icon
         name={iconRight}
         size="small"
+        className="icon-right"
```

Comment:
although this is a possible way to add css, let's stay consistent and define a new styled component with the different styles.

for instance 
```js
const StyledIcon = styled(Icon)`

/* ...css here */

`;
```

and then 

```js
 <StyledIcon
        name={iconRight}
        size="small"
 ```

Reply:
Makes sense, done.